apps/views.py

```diff
@@ -281,10 +281,14 @@ def update_resource(self, request, appinstance, current_release_name):
         appinstance.table_field.update({"url": new_url})
         if new_release_name and current_release_name != new_release_name:
             # This handles the case where a user creates a new subdomain, we must update the helm release aswell
-            _ = delete_and_deploy_resource.delay(appinstance.pk, new_release_name)
-        else:
-            # Otherwise, we update the resources in the same helm release
-            _ = deploy_resource.delay(appinstance.pk, "update")
```

Comment:
I haven't tested this yet but .delay() here is supposed to add this as a task in the queue for celery and not using it might just run it in the studio pod instead. Would be good to test if this is the case

Reply:
It adds the task to celery anyways, but the delay function just applies it asynchronously. But we should add delay here again. This is a mistake.